packages/ui/src/__stories__/ButtonMigration/Variants.tsx

```diff
@@ -1,91 +1,9 @@
-import type { ComponentProps } from 'react'
-import { Button, ButtonV2, Table } from '../../components'
-
-const MOCK_ONCLICK = () => {}
-
-const VARIANT_MATCH: Record<
-  Required<ComponentProps<typeof Button>>['variant'],
-  {
-    variant: Required<ComponentProps<typeof ButtonV2>>['variant']
-    sentiment: Required<ComponentProps<typeof ButtonV2>>['sentiment']
-  }
-> = {
-  info: { variant: 'filled', sentiment: 'info' },
-  'info-bordered': { variant: 'outlined', sentiment: 'info' },
-  link: { variant: 'ghost', sentiment: 'primary' },
-  primary: { variant: 'filled', sentiment: 'primary' },
-  'primary-bordered': { variant: 'outlined', sentiment: 'primary' },
-  'primary-soft-bordered': { variant: 'outlined', sentiment: 'neutral' },
-  secondary: { variant: 'filled', sentiment: 'neutral' },
-  'secondary-bordered': { variant: 'outlined', sentiment: 'neutral' },
-  success: { variant: 'filled', sentiment: 'success' },
-  'success-bordered': { variant: 'outlined', sentiment: 'success' },
-  'success-soft-bordered': { variant: 'outlined', sentiment: 'neutral' },
-  transparent: { variant: 'ghost', sentiment: 'neutral' },
-  warning: { variant: 'filled', sentiment: 'danger' },
-  'warning-bordered': { variant: 'outlined', sentiment: 'danger' },
-  'warning-soft-bordered': { variant: 'outlined', sentiment: 'neutral' },
-}
+import ButtonActionMigrationImg from '../assets/button-action-migration.png'
+import ButtonMigrationImg from '../assets/button-migration.png'
```

Comment:
The migration page on storybook doesn't seems to work anymore, I have a white page